pkg/test/config/toolchainconfig.go

```diff
@@ -579,6 +579,19 @@ func (o TiersOption) DurationBeforeChangeTierRequestDeletion(value string) Tiers
 	return o
 }
 
+func (o TiersOption) FeatureToggle(name string, weight *uint) TiersOption {
+	o.addFunction(func(config *toolchainv1alpha1.ToolchainConfig) {
+		if config.Spec.Host.Tiers.FeatureToggles == nil {
+			config.Spec.Host.Tiers.FeatureToggles = make([]toolchainv1alpha1.FeatureToggle, 0)
```

Comment:
Shouldn't the default be 100?

Reply:
oops, was too quick reading it, you're just initializing the array 🙂

Reply:
yeah.. I don't really think we need to initialize the array at all. You can append to a nil slice but just to be 100% safe... :)